apps/cow-fi/app/(main)/legal/cowswap-terms/page.tsx

```diff
@@ -711,8 +711,8 @@ export default function Page() {
               Must be a real user who conducted trades on CoW Swap on the Base Network during February 2025 and must:
             </p>
             <ul>
-              <li>be minimum 18 years of age.</li>
-              <li>Not be an employee, affiliate, or immediate family member of the Organizer.</li>
+              <li>be minimum 18 years of age;</li>
+              <li>not be an employee, affiliate, or immediate family member of the Organizer.</li>
```

Comment:
wouldn't you want to do the opposit, start each bullet point with capital letter?

For example, in line 761 a bullet list used capital letters

Reply:
Fixed to use capitals like it is used in listings at the top part of the document: 
![image](https://github.com/user-attachments/assets/a997776c-8df7-4f81-8a1c-eddeb28b0640)
![image](https://github.com/user-attachments/assets/d7540172-5f8d-41e0-aed1-19335fa00615)